app/src/docs/v1.api-spec.yaml

```diff
@@ -17,30 +17,348 @@ security:
   - BearerAuth: []
     OpenID: []
 paths:
-  /hello:
+  '/':
     get:
-      summary: Returns hello world result
-      operationId: getHello
+      summary: List all user accessible objects
+      description: >-
+        This endpoint returns a list of all user accessible objects.
+      operationId: GetAccessibleObjects
+      tags:
+        - Object
+      responses:
+        '200':
+          description: Returns the object information
```

Comment:
'returns an array of object id's'

Reply:
https://github.com/bcgov/common-object-management-service/pull/9/files#r803069575

According to Jeremy's comment I will be leaving this as is and be changing these endpoints accordingly.
POST /object
GET /object/{objId}/versions